vars/runCleanupResource.groovy

```diff
@@ -21,6 +21,7 @@ def call(Map params, String region){
     export SCT_POST_BEHAVIOR_DB_NODES="${params.post_behavior_db_nodes}"
     export SCT_POST_BEHAVIOR_LOADER_NODES="${params.post_behavior_loader_nodes}"
     export SCT_POST_BEHAVIOR_MONITOR_NODES="${params.post_behavior_monitor_nodes}"
+    export SCT_POST_BEHAVIOR_K8S_cluster="${params.post_behavior_k8s_cluster}"
```

Comment:
Env vars are expected to be in uppercase completely, not partially.

Reply:
fixed

Reply:
Dont see it is